tests/processor/test_orchestrator.py

```diff
@@ -52,7 +52,7 @@ class OrchestratorTest(MasuTestCase):
 
     def setUp(self):
         self.mock_accounts = []
-        self.num_accounts = range(1, random.randint(1,20))
+        self.num_accounts = range(1, random.randint(5,20))
         for _ in self.num_accounts:
```

Comment:
For future reference, since num_accounts is only referenced here, it doesn't strictly need to be a `self.` var. This is certainly personal preference speaking, but `for _ in range(...):` is a fine for loop.

Reply:
Correction: It _was_ being used elsewhere, until I realized I was testing the wrong thing and changed it. So, yes, you're right. Using self is unnecessary here. I'll fix it.